repo/packages/R/redis/2/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "3.0",
+ "name": "redis",
+ "version": "4.0-0.0.1",
+ "maintainer": "joerg@mesosphere.io",
+ "description": "This is a single redis container, which is NOT suited for HA setups. Redis is a popular in-memory data structure store, used as database, cache and message broker. See documentation for more details: https://github.com/dcos/examples/tree/master/redis",
+ "website": "http://www.redis.io",
+ "framework": false,
+ "tags": ["mesosphere", "service", "redis", "storage"],
+ "licenses": [
+   {
+     "name": "BSD",
+     "url": "http://redis.io/topics/license"
+   }
+ ],
+ "website": "http://redis.io",
+ "postInstallNotes": "Redis installed!",
+ "preInstallNotes": "This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Redis requires a single node with 300MB of RAM and 0.1 CPUs.",
```

Comment:
This is incorrect. 1 cpu, 1024 MB according to config.json.

Reply:
You can still configure less.